tools/react-app-template/package.json

```diff
@@ -0,0 +1,122 @@
+{
+    "name": "@gooddata/react-app-template",
+    "version": "1.0.0",
+    "description": "GoodData Template for React App",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/gooddata/gooddata-ui-sdk.git",
+        "directory": "tools/react-app-template"
+    },
+    "author": "GoodData Corporation",
+    "private": true,
+    "license": "MIT",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "sideEffects": [
+        "*.css",
+        "*.svg"
+    ],
+    "files": [
+        "dist/**/*.js",
+        "dist/**/*.json",
+        "dist/**/*.d.ts",
+        "dist/**/*.map",
+        "dist/**/*.svg"
+    ],
+    "scripts": {
+        "clean": "rm -rf ci dist esm coverage *.log styles/css && jest --clearCache",
+        "build": "webpack --mode production",
+        "start": "webpack-cli serve",
+        "test": "jest --watch --passWithNoTests",
+        "test-once": "jest --maxWorkers=${JEST_MAX_WORKERS:-'45%'} --passWithNoTests",
+        "test-ci": "JEST_JUNIT_OUTPUT=./ci/results/test-results.xml jest --ci --config jest.ci.js --passWithNoTests",
+        "eslint": "eslint -c .eslintrc.js --ext ts,tsx src/",
+        "eslint-ci": "mkdir -p ./ci/results && eslint -f checkstyle -o ci/results/eslint-results.xml -c .eslintrc.js --ext ts,tsx src/",
+        "prettier-check": "prettier --check \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
+        "prettier-write": "prettier --write \"{src,test}/**/*.{ts,tsx,json,scss,md,yaml,html}\"",
+        "dep-cruiser": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "dep-cruiser-ci": "depcruise --validate .dependency-cruiser.js --output-type err-long src/",
+        "validate": "npm run dep-cruiser && npm run eslint && npm run prettier-check",
+        "validate-ci": "npm run dep-cruiser-ci && npm run eslint-ci && npm run prettier-check",
+        "refresh-md": "gdc-catalog-export"
+    },
+    "dependencies": {
+        "@gooddata/sdk-backend-base": "^8.12.0-alpha.62",
```

Comment:
Will we somehow keep `yarn.lock` and `package-lock.json` files for this? Or will it always force the latest?
With the lockfiles this could enable users to create apps with older SDK versions (which can be useful for Tiger users).

Reply:
That's a very good point... But how to integrate this properly with the release? 🤔 Remove carets as a part of the CLI command? 😄 Add script for it as a part of the release pipeline? Split the release into two jobs? 😕

Reply:
Should be solved with: https://gooddata.atlassian.net/browse/RAIL-4873